docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.install

```diff
@@ -246,3 +246,106 @@ function hs_paragraph_types_update_10006() {
     }
   }
 }
+
+/**
+ * Set new config for collection color wells.
+ */
+function hs_paragraph_types_update_10007(&$sandbox) {
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  if (empty($sandbox['ids'])) {
+
+    // Import all config before updating existing paragraphs.
+    _hs_paragraph_types_well_color_config();
+
+    // Get list of collection paragraphs.
+    $sandbox['ids'] = $paragraph_storage->getQuery()
+      ->accessCheck(FALSE)
+      ->condition('type', ['hs_collection', 'hs_priv_collection'], 'IN')
+      ->execute();
+    $sandbox['total'] = count($sandbox['ids']);
+  }
+
+  $paragraph_ids = array_splice($sandbox['ids'], 0, 10);
+
+  // Set new field default for existing hs_collection paragraphs.
+  foreach ($paragraph_storage->loadMultiple($paragraph_ids) as $paragraph) {
+    /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+    if (
+      !$paragraph->hasField('field_paragraph_style') ||
+      !$paragraph->hasField('field_bg_color') ||
+      !$paragraph->hasField('field_bg_color_width')) {
+      continue;
+    }
+
+    // Only set new field if old field had data.
+    if ($paragraph->get('field_paragraph_style')->value) {
+      $paragraph->set('field_bg_color', 'default');
+    }
+    if ($paragraph->get('field_paragraph_style')->value == 'well-full-width') {
+      $paragraph->set('field_bg_color_width', 'full_width');
+    }
+    else {
+      $paragraph->set('field_bg_color_width', 'limited_width');
+    }
```

Comment:
```suggestion
    // Only set new field if old field had data.
    $field_paragraph_style = $paragraph->get('field_paragraph_style')->value;
    if ($field_paragraph_style) {
      $paragraph->set('field_bg_color', 'default');
    }
    $bg_color_width = 'limited_width';
    if ($field_paragraph_style == 'well-full-width') {
      $bg_color_width = 'full_width';
    }
    $paragraph->set('field_bg_color_width', $bg_color_width);
```

1. Instead of making two calls to `get->('field_paragraph_style')` store the result.
2. Probably just a preference, but instead of if/else, set a default value for the `bg_color_width` and change it with an if condition.

Reply:
Fixed!